### DISCUSSION TITLE
When are you planing to upgrade arduino version from 4.4.99 to 5.1 ?

### DISCUSSION THREAD

**Discussion by IvanViale** [espressif/arduino-esp32#8613] [General]
Hi, i have a stack problem using ble and wifi at same time, i saw a lot of information on internet and i found this article that tell me that the issue is solved on IDF version 5.1 but arduino is stuck on 4.4.99.

https://github.com/espressif/arduino-esp32/issues/6915

Now i'm using arduino ide, when are you planing to upgrade the version to 5.1? Otherwise i need to rewrite all with ESP-IDF or did you have another solution?

Thanks

**Comment by me-no-dev** [comment]
alpha release based on 5.1 will be out next week

**Reply by IvanViale** [reply]
Wonderful !!! Can i download it directly by Arduino Ide or i need to download it in another place?

**Reply by IvanViale** [reply]
Sorry, but I can find only V2.0.12 based on ESP-IDF v4.4.5, where i can find the alpha release with v5.1? Thanks

**Reply by me-no-dev** [reply]
You will find it when we release it.

**Reply by IvanViale** [reply]
Ok, thanks

**Reply by IvanViale** [reply]
Hi, sorry for my insistence but I don't want to have misunderstood, I saw that update 2.0.13 has been released which has the ESP-IDF version v4.4.5, 3.0.0 with version v5.1.0 will also be released soon or did I misunderstand? I have a project that I will start selling in 2 weeks and I have the problem with the bluetooth that restart the device because I use it together with wifi and I need to know if I can solve it using the Arduino IDE or if I have to change the IDE. 

Thank you very much !!!

**Reply by me-no-dev** [reply]
only 3.0.0-alpha will be released. There is still long to go for stable 3.0.0, so I suggest you do not even think about going into production with 3.0.0 in it's current state. Plus there are some API changes that will cause compatibility issues.
